### ISSUE TITLE
force_split and no_gaps_when_only prevents dragging windows

### ISSUE THREAD

**Issue by ProMix0** [hyprwm/Hyprland#1904]
## Steps to reproduce
Example configuration:
```
dwindle {
    force_split = 2 # setting to 1 also produce bug
    no_gaps_when_only = on
}
```

## Expected outcome
Normal windows moving

## Noted outcome
Window can't be moved from right to left (from left to right when force_split = 1) when dragging it with mouse. It's actual for two and more windows

**Comment by ProMix0** [comment]
By the way, it's also prevents to move tabs in Yandex Browser (chrome-based) when only window opened

**Comment by vaxerski** [comment]
that is expected behavior though, you _forced_ the split...?

**Comment by vaxerski** [comment]
fixed in a4c120d6081ee35b7997afdec451480048242d06
